wiki/Options/en.md

```diff
@@ -529,609 +166,141 @@ Note: Your current playing song on your jukebox will be used.
-<colgroup>
-<col width="12%" />
-<col width="24%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th><p>Name</p></th>
-<th><p>Effect</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Key Count</p></td>
-<td><p>Normal settings for 1K ~ 9K, 5K ~ 9K Co-op (same as osu!mania editor's possible K options).</p></td>
-</tr>
-<tr class="even">
-<td><p>Co-op Layout</p></td>
-<td><p>Settings after applying Co-op mod (currently apply to 1K ~ 9K. 10K ~ 18K is still in progress)</p></td>
-</tr>
-<tr class="odd">
-<td><p>Column Style<br />
-[6K and 8K only]</p></td>
-<td><table>
-<thead>
-<tr class="header">
-<th><p>CS</p></th>
-<th><p>6K</p></th>
-<th><p>8K</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Normal</p></td>
-<td><div class="figure">
-<img src="CS_6K_N.jpg" title="CS_6K_N.jpg" alt="CS_6K_N.jpg" width="150" />
-<p class="caption">CS_6K_N.jpg</p>
-</div></td>
-<td><div class="figure">
-<img src="CS_8K_N.jpg" title="CS_8K_N.jpg" alt="CS_8K_N.jpg" width="200" />
-<p class="caption">CS_8K_N.jpg</p>
-</div></td>
-</tr>
-<tr class="even">
-<td><p>Right</p></td>
-<td><div class="figure">
-<img src="CS_6K_R.jpg" title="CS_6K_R.jpg" alt="CS_6K_R.jpg" width="150" />
-<p class="caption">CS_6K_R.jpg</p>
-</div></td>
-<td><div class="figure">
-<img src="CS_8K_R.jpg" title="CS_8K_R.jpg" alt="CS_8K_R.jpg" width="200" />
-<p class="caption">CS_8K_R.jpg</p>
-</div></td>
-</tr>
-<tr class="odd">
-<td><p>Left</p></td>
-<td><div class="figure">
-<img src="CS_6K_L.jpg" title="CS_6K_L.jpg" alt="CS_6K_L.jpg" width="150" />
-<p class="caption">CS_6K_L.jpg</p>
-</div></td>
-<td><div class="figure">
-<img src="CS_8K_L.jpg" title="CS_8K_L.jpg" alt="CS_8K_L.jpg" width="200" />
-<p class="caption">CS_8K_L.jpg</p>
-</div></td>
-</tr>
-</tbody>
-</table></td>
-</tr>
-<tr class="even">
-<td><p>Bind alternate special keys</p></td>
-<td><p>[When using right/left column style] Bind the alternate special key.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Vertically flip playfield<br />
-(DDR Style)</p></td>
-<td><p>Move the judgement area and keynotes above the screen instead of the usual below the screen. Beatmap's design and key-bindings stay as it is.</p></td>
-</tr>
-<tr class="even">
-<td><p>Show Judgement line</p></td>
-<td><p><a href="Media:OP_J_Line.jpg" title="wikilink">Make Judgement Line slightly &quot;brighter&quot;.</a></p></td>
-</tr>
-<tr class="odd">
-</tr>
-</tbody>
-</table>
+| Name                                  | Effect                                                                                          |
+|---------------------------------------|-------------------------------------------------------------------------------------------------|
+| Key Count                             | Normal settings for 1K ~ 9K, 5K ~ 9K Co-op                                                      |
+| Co-op Layout                          | Settings after applying co-op mod (currently apply to 1K ~ 9K. 10K ~ 18K is still in progress). |
```

Comment:
(Am I reading this correctly??) needs update; you can change the key layout for 10 and up